gen_cert.py

```diff
@@ -2065,6 +2065,12 @@ def _generate_v4_certificate(
                             continue
                         draw_template_element(self, element, attributes, page)
 
+        # Render any badges
```

Comment:
While full z-index may be overkill, a reasonable compromise could be just `top` and `bottom`. Since we've identified these two use cases (0-1's on bottom, watermark on top), I'd rather implement this completely, instead of finding ourselves coming back and editing the renderer yet again, since that's why we wrote v4.

```yaml
Course/Triple/Here:
    # ...
    flair:
        - type: image
          order: bottom  # default is top
        - type: image
```

```python
# initial setup goes here
for item in flair:
    if item.order == 'bottom':
        # render here

# currently existing rendering logic

for item in flair:
    if item.get('order', 'top') == 'top':
        # render here
```